### DISCUSSION TITLE
Accidentally disabled the USB port

### DISCUSSION THREAD

**Discussion by Moon1moon** [DavidS95/Smokeless_UMAF#34] [Q&A]
Accidentally disabled the USB port over the app

I can't boot to the app because the laptop no longer reads the USB stick. 

It doesn't read the USB C port either. I reset the classic BIOS to its default settings, but the USB settings did not change.

Is there any way I can fix this without physically reaching the Motherboard?

Thanks

**Comment by ff66theone** [comment]
Hello
I hope I'm not too late, but you might be able to install Smokeless UMAF on the hard disk's ESP partition and boot it using a bootloader like GRUB2.
That's what I did for Smokeless and CSMWrapper and both work reasonably well using this method.
